### PR TITLE
wellcome page first design

### PR DESCRIPTION
**Pull request by YunusEmreBEYAZ**

### CODE REVIEW

**Review by A-Ghoujal — APPROVED** [review]

**Comment by sima-milli** [comment]
We can wait until the other PR is merged and them merge this one because the other one should have a fix for the quiz

**Review by dakravchenko — APPROVED** [review]

**Review by A-Ghoujal — APPROVED** [review]

**Review by sima-milli — APPROVED** [review]

**Pull request merged**
